src/Core/Grid/Presenter/GridPresenter.php

```diff
@@ -81,7 +81,7 @@ public function present(GridInterface $grid)
         ];
 
         $this->hookDispatcher->dispatchWithParameters('action' . $definition->getId() . 'GridPresenterModifier', [
-            'presentedGrid' => $presentedGrid,
+            'presented_grid' => $presentedGrid,
             'grid' => $grid,
```

Comment:
do we actually need `$grid` as parameter here?

Reply:
no, I guess it's a bad rebase :/ only Grid presenter should be available here.